core/tests/gae_suite.py

```diff
@@ -80,18 +80,6 @@ def create_test_suites(test_target=None):
 
 def main():
     """Runs the tests."""
-
-    def _iterate(test_suite_or_case):
```

Comment:
This function is not used anywhere and the backend tests seem to run fine without it that's why I removed it